Activities/Database/UiPath.Database.Activities.Design/DesignerMetadata.cs

```diff
@@ -39,14 +39,14 @@ public void Register()
             builder.AddCustomAttributes(typeof(BulkUpdate), contentAttr);
 
             // DisplayName attribute
-            builder.AddCustomAttributes(typeof(DatabaseConnect), new DisplayNameAttribute(Resources.Connect));
-            builder.AddCustomAttributes(typeof(DatabaseDisconnect), new DisplayNameAttribute(Resources.Disconnect));
-            builder.AddCustomAttributes(typeof(DatabaseTransaction), new DisplayNameAttribute(Resources.StartTransaction));
-            builder.AddCustomAttributes(typeof(ExecuteNonQuery), new DisplayNameAttribute(Resources.ExecuteNonQuery));
-            builder.AddCustomAttributes(typeof(ExecuteQuery), new DisplayNameAttribute(Resources.ExecuteQuery));
-            builder.AddCustomAttributes(typeof(InsertDataTable), new DisplayNameAttribute(Resources.Insert));
-            builder.AddCustomAttributes(typeof(BulkInsert), new DisplayNameAttribute(Resources.BulkInsert));
-            builder.AddCustomAttributes(typeof(BulkUpdate), new DisplayNameAttribute(Resources.BulkUpdate));
```

Comment:
Validate that this resources (old activity names) are no longer used and remove them from RESX (no longer required - cluttering the RESX)